### ISSUE TITLE
Indicate that tab will be closed soon

### ISSUE THREAD

**Issue by qzb** [qzb/dustman#14]
When tab is about to close it's favicon should be changed.

**Comment by Sanqui** [comment]
Alternatively, the title text could grey out.  AutoClose Tabs did this.

**Comment by qzb** [comment]
Sounds like a good idea, but it's not future proof. It can be achieved only with API which will be deprecated soon.

**Comment by luckydonald** [comment]
Having a counter updating in the title? 
But that feels way too retro (even [**`aesthetic`**](http://knowyourmeme.com/memes/aesthetic))

**Comment by luckydonald** [comment]
Seeing other plugins, a common way to display status is the favicon:
![image](https://cloud.githubusercontent.com/assets/2737108/16154593/d5bba61e-34ab-11e6-9cb3-e681c5e10bd4.png)
TabMixPlus attaches little icons to the icon of the webpage, TileTab uses colored borders.
Also I believe animated icons (gif?) can be displayed.

**Comment by qzb** [comment]
Yep, I am thinking about something like this. It could be tricky to achieve but it will be looking nice and should be future proof.

**Comment by mafjmafj** [comment]
once we got this ... make actual closing optional... I would be ok closing tabs myself an indication I did not use one for a while would be great help.

**Comment by luckydonald** [comment]
@mafjmafj that should be a separate issue.

**Comment by luckydonald** [comment]
Not sure, can add-ons use a canvas while not being in the foreground?
There are functions to draw a given image onto a canvas, and also functions to get the full content of the canvas as `data:` url.
